proposal.html

```diff
@@ -1,14 +1,14 @@
 <pre class=metadata>
-title: Array.prototype.flatMap &amp; Array.prototype.flatten
+title: Array.prototype.flatMap &amp; Array.prototype.smoosh
```

Comment:
I am a proficient non-native, English speaker and still I did not know the existence of the word `smoosh` until today.